### ISSUE TITLE
Feature Request: Add method to return remaining time on the interval

### ISSUE THREAD

**Issue by oxo42** [Aasim-A/AsyncTimer#17]
I want to be able to report how much time is left to the next interval on a timer.  I would like to write code

```c++
unsigned long remaining = t.getRemaining(timerId);
Serial.print("Time remaining: ");
Serial.println(remaining);
```

**Comment by Aasim-A** [comment]
Hi mate, that would be a nice addition, I'll be adding it as soon as I can 👍

**Comment by Aasim-A** [comment]
Fixed by #18
